master/usage/calicoctl.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Installing and Configuring calicoctl
+---
+
+This document outlines how to install and configure calicoctl which is the
+primary tool for viewing, manipulating, and creating Calico objects on the
+command line.
+
+### Location, location, location
+
+The majority of calicoctl operations only need access to the datastore that
```

Comment:
I think the point that this is getting at but isn't clearly stating is:

Calicoctl's primary function is to manipulate state stored in etcd. As such, it can run from any host with network access to the cluster's etcd.